### ISSUE TITLE
Allow to define multiple engines per note

### ISSUE THREAD

**Issue by antonpodolsky** [wix/quix#312]

**Comment by frolovv** [comment]
@antonpodolsky 
can you give an example? 
what is `note type` and what is `engine` in your proposal?
how does it differ from existing approach? 
will it be used for executing the same note on different clusters, prod vs dev for example?

**Comment by antonpodolsky** [comment]
Yes, the idea is to execute same note on different clusters, without duplicating the note.
